circle.yml

```diff
@@ -0,0 +1,44 @@
+machine:
+  python:
+    version: 2.7.10
+
+dependencies:
+  cache_directories:
+    - ../examples/
+
+  pre:
+    - sudo pip install --upgrade pip
+    - sudo pip install traitlets pexpect pickleshare simplegeneric
+    - cd .. && wget https://pypi.python.org/packages/source/i/ipython/ipython-4.1.1.tar.gz && tar -xzf ipython-4.1.1.tar.gz && cd ipython-4.1.1 && sudo python setup.py install
+    - sudo pip install sphinx==1.2.3 pygraphviz matplotlib Pillow neo scipy numpy
+    - cd .. && git clone https://github.com/sphinx-gallery/sphinx-gallery.git && cd sphinx-gallery && sudo pip install -r requirements.txt && sudo python setup.py develop
+    - sudo pip install sphinx_bootstrap_theme
+    - sudo pip install numpy --upgrade
+    - sudo apt-get install libvtk5-dev python-vtk python-configobj
+    - sudo pip install mayavi
+    - sudo pip install pysurfer
+    - /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1400x900x24 -ac +extension GLX +render -noreset
+    - sudo pip install scikit-learn nilearn
+    - sudo python setup.py install
+    - sudo make sample_data
```

Comment:
do we have to do everything with `sudo`, or are you just doing that now for debugging simplicity?